src/app/tests/suites/certification/Test_TC_MC_1_10.yaml

```diff
@@ -43,49 +43,65 @@ tests:
           constraints:
               type: uint16
 
+    #Issue 10988
     - label: "Read the global attribute: AttributeList"
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AttributeList"
       response:
+          value: [0, 1, 2, 3, 4, 5, 6, 7, 65528, 65529, 65531, 65533]
           constraints:
               type: list
 
     #issue #11053 disabled steps below Global attributes missing from YAML framework
-    - label: "Read the global attribute: EventList"
-      disabled: true
-      command: "readAttribute"
-      attribute: "EventList"
-      response:
-          constraints:
-              type: list
+    - label:
+          "Read EventList attribute from the DUT and Verify that the DUT
+          response provides a list of supported events."
+      verification: |
+          Not implemented in chip-tool
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
 
+    #Issue 10988
     - label: "Read the global attribute: AcceptedCommandList"
-      disabled: true
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AcceptedCommandList"
       response:
+          value: [0, 1]
           constraints:
               type: list
 
+    #Issue 17459
     - label: "Read the global attribute: GeneratedCommandList"
-      disabled: true
+      PICS: PICS_SKIP_SAMPLE_APP
```

Comment:
```suggestion
      PICS: SKIP_SAMPLE_APP
```

Reply:
josh(@krypton36 ) told me to use PICS_SKIP_SAMPLE_APP

Reply:
Well we are missing consistency 
Some use prefix PICS_ some not 
if you guys choose one or the other that means you should do the update everywhere .... I would picks the without prefix one

Reply:
@krypton36 what do you think ?

Reply:
I have no problem using SKIP_SAMPLE_APP. It is pretty explicit, that it is a protocol implementation conformance statement. @manjunath-grl Can you please update it without the PICS prefix? I filed an issue to remove the PICS prefix to any other [here](https://github.com/project-chip/connectedhomeip/issues/17643).